src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -101,6 +101,7 @@ public FileStatusList()
                 Name = "sortListByContextMenuItem"
             };
 
+            tsmiMove.Image = Properties.Images.Renamed.AdaptLightness();
```

Comment:
Renamed icon is so far only used to rename branches
![Renamed](https://github.com/user-attachments/assets/f1d10359-6153-4344-a0be-678b5c84fc44)

Renamed files are seen in thediff tab. Should this use that icon instead?
(also need to be adapted)
![FileStatusRenamed](https://github.com/user-attachments/assets/068d0c12-4a07-477c-8f1e-6502c9ef0b36)

Reply:
Yes, in the context of file status, this (rather unclear) icon seems to be the better choice.